packages/lodestar/src/network/encoders/request.ts

```diff
@@ -47,16 +47,34 @@ export function eth2RequestDecode(
         }
         if(sszDataLength === null) {
           sszDataLength = decode(chunk.slice());
+          if (decode.bytes > 10) {
+            logger.error(`eth2RequestDecode: Invalid number of bytes for protobuf varint ${decode.bytes}` +
+            `, method ${method}`);
+            yield RpcResponseStatus.ERR_INVALID_REQ;
```

Comment:
I don't think it's wise to yield that, body can be just number so it's confusing. Just print warn message and break so request doesn't get propagated

Reply:
I see your point, but we have to send `ERR_INVALID_REQ` status to the client in this case ... so definitely have to come up with a different way